### DISCUSSION TITLE
Add another shlink instances

### DISCUSSION THREAD

**Discussion by buzzing-bee** [shlinkio/shlink#2128] [Help wanted] [answered]
I am successfully install and configure a shlink instances using the web (php) version.
If I want to add another instance with different API Key, how do I do it?

Let say the current instance is installed in folder "shlink-1", am I correct that I need to install and configure the second instance in another folder ("shlink-2" for example) ??

**Comment by acelaya** [comment, marked as answer]
Yes, that's correct.

**Comment by buzzing-bee** [comment]
thanks @acelaya , works like a charm
